util/string.c

```diff
@@ -4,7 +4,7 @@
 #include <stdint.h>
 #include <ctype.h>
 
-#ifdef __GNUC__
+#if defined(__GNUC__) && !defined(__llvm__)
```

Comment:
Why `__llvm__` rather than `__clang__`? The latter is the normal thing to use...